### DISCUSSION TITLE
不用外部插帧，仅使用tscale的用户，需要设置`hr-seek-framedrop=no`吗

### DISCUSSION THREAD

**Discussion by yeezylife** [hooke007/mpv_PlayKit#184] [Q&A 问答] [answered]
我只开启了
'''video-sync=display-resample
interpolation=yes
tscale=spline16'''

然后我读了读手册里关于'hr-seek-framedrop'的描述，只说了"if you're using video filters which modify timestamps or add new frames"可能导致跳帧，我虽然不用外部的插帧，但是tscale好像也改变了帧数/生成了一些中间帧呀。

是不是不用外部插帧，仅使用tscale的用户，也设置 ‘hr-seek-framedrop=no’ 比较好？

**Comment by hooke007** [comment, marked as answer]
hr-seek-framedrop=no 可能会导致精确跳转更慢，我用它是为了防止修改源帧率类的滤镜跳转后失去同步用的，跳转时我只在乎音视频对齐。（就算丢帧难道不也是为了快速对齐音频然后播放吗）
tscale不修改源帧率，它混成的帧只影响最后输出阶段的帧率，不修改源帧率的情况下跳转也不存在失去同步的问题。

**Reply by yeezylife** [reply]
> 跳转时我只在乎音视频对齐。（就算丢帧难道不也是为了快速对齐音频然后播放吗）

原来是这回事，我以为丢了帧，后面的画面和声音就一直错开了。 确实跳转的时候我也只在乎音视频对齐。

不过我后来决定还是开着了，只要这个选项能减少出错的可能。一来我肉眼试了试没感觉出快进的速度有什么差异，二来也不怎么快进，也就跳个OP。。

**Reply by hooke007** [reply]
除开跳转，几乎所有播放器都默认允许播放时丢帧以对齐音频，只要不瞎搞也不存在你担心的问题
